tests/Games/NexusMods.Games.StardewValley.Tests/Analyzers/SMAPIManifestAnalyzerTests.cs

```diff
@@ -1,107 +0,0 @@
-using System.Diagnostics.CodeAnalysis;
-using System.Text.Json;
-using FluentAssertions;
-using NexusMods.FileExtractor.FileSignatures;
-using NexusMods.Games.StardewValley.Analyzers;
-using NexusMods.Games.StardewValley.Models;
-using NexusMods.Games.TestFramework;
-
-namespace NexusMods.Games.StardewValley.Tests.Analyzers;
-
-[SuppressMessage("ReSharper", "InconsistentNaming")]
-public class SMAPIManifestAnalyzerTests : AFileAnalyzerTest<StardewValley, SMAPIManifestAnalyzer>
-{
-    public SMAPIManifestAnalyzerTests(IServiceProvider serviceProvider) : base(serviceProvider) { }
-
-    [Fact]
-    public void Test_FileTypes()
-    {
-        FileAnalyzer.FileTypes.Should().ContainSingle(x => x == FileType.JSON);
-    }
-
-    [Fact]
-    public async Task Test_Analyze()
-    {
-        var expected = new SMAPIManifest
-        {
-            Name = Guid.NewGuid().ToString("N"),
-            Version = new SMAPIVersion
-            {
-                MajorVersion = 1,
-                MinorVersion = 2,
-            },
-            UniqueID = Guid.NewGuid().ToString("N"),
-            MinimumApiVersion = new SMAPIVersion
-            {
-                MajorVersion = 3,
-                MinorVersion = 12
-            }
-        };
```

Comment:
Are we sure we want to delete all these tests? Wouldn't some of these still be useful to notice regressions in SMAPI manifest parsing?

Reply:
Those would then show up in regressions in the installer library

Reply:
We're going to need to do a pass to fix SMAPI handling, really to enable it for the first time.

Reply:
created #645